src/Microsoft.Extensions.Configuration.AzureKeyVault/KeyVaultClientWrapper.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.Azure.KeyVault;
+
+namespace Microsoft.Extensions.Configuration.AzureKeyVault
+{
+    /// <inheritdoc />
+    public class KeyVaultClientWrapper : IKeyVaultClient
+    {
+        private readonly KeyVaultClient _keyVaultClientImplementation;
+
+        /// <summary>
+        /// Creates a new instance of <see cref="KeyVaultClientWrapper"/>.
+        /// </summary>
+        /// <param name="keyVaultClientImplementation">The <see cref="KeyVaultClient"/> instance to wrap.</param>
+        public KeyVaultClientWrapper(KeyVaultClient keyVaultClientImplementation)
+        {
+            _keyVaultClientImplementation = keyVaultClientImplementation;
+        }
+
+        /// <inheritdoc />
+        public Task<ListSecretsResponseMessage> GetSecretsAsync(string vault)
+        {
+            return _keyVaultClientImplementation.GetSecretsAsync(vault);
+        }
+        /// <inheritdoc />
+        public Task<Secret> GetSecretAsync(string secretIdentifier)
+        {
+            return _keyVaultClientImplementation.GetSecretAsync(secretIdentifier);
+        }
+        /// <inheritdoc />
```

Comment:
AFAIK Intellisense doesn't work with members that have `<inheritdoc>` so we shouldn't be using it. But this information may be outdated.